api/PerformanceResourceTiming.json

```
@@ -1000,9 +1008,21 @@
             "web-features:resource-timing"
           ],
           "support": {
-            "chrome": {
-              "version_added": "43"
-            },
+            "chrome": [
+              {
+                "version_added": "133"
+              },
+              {
+                "version_added": "115",
+                "partial_implementation": true,
+                "notes": "`responseStart` used interim response instead of final response ([chromestatus](https://chromestatus.com/feature/5158830722514944))."
```

Given what the spec says:

> The responseStart getter steps are to return [this](https://webidl.spec.whatwg.org/#this)'s [firstInterimResponseStart](https://w3c.github.io/resource-timing/#dom-performanceresourcetiming-firstinterimresponsestart) if it is not 0; Otherwise [this](https://webidl.spec.whatwg.org/#this)'s [finalResponseHeadersStart](https://w3c.github.io/resource-timing/#dom-performanceresourcetiming-finalresponseheadersstart).

Does this note mean that `responseStart` always returned `firstInterimResponseStart`, even if it's 0?

This would contradict `firstInterimResponseStart` always returning the value of `finalResponseHeadersStart`, unless `responseStart` always returned the _expected_ value of `firstInterimResponseStart`, and the _actual_ value of `firstInterimResponseStart` being the (expected and? actual) value of `finalResponseHeadersStart`.

---

Oh this not is wrong!

```suggestion
                "notes": "`responseStart` used final response even if interim response was available ([chromestatus](https://chromestatus.com/feature/5158830722514944))."
```